File: java/src/main/java/com/meteroid/models/SubscriptionFeeBillingPeriodEnum.java
// This file is @generated
package com.meteroid.models;

import com.fasterxml.jackson.annotation.JsonValue;
import com.meteroid.Utils.ToQueryParam;

public enum SubscriptionFeeBillingPeriodEnum implements ToQueryParam {
    ONE_TIME("ONE_TIME"),
    MONTHLY("MONTHLY"),
    QUARTERLY("QUARTERLY"),
    SEMIANNUAL("SEMIANNUAL"),
    ANNUAL("ANNUAL");
    private final String value;

    SubscriptionFeeBillingPeriodEnum(String value) {
        this.value = value;
    }

    @JsonValue
    public String getValue() {
        return this.value;
    }

    @Override
    public String toQueryParam() {
        return this.value;
    }
}

File: python/meteroid/models/subscription_fee_billing_period_enum.py
# this file is @generated
import enum


class SubscriptionFeeBillingPeriodEnum(str, enum.Enum):
    ONE_TIME = "ONE_TIME"
    MONTHLY = "MONTHLY"
    QUARTERLY = "QUARTERLY"
    SEMIANNUAL = "SEMIANNUAL"
    ANNUAL = "ANNUAL"

    def __str__(self) -> str:
        return str(self.value)
